encryption/client_side_encryption.py

```diff
@@ -0,0 +1,39 @@
+import hashlib
+import aws_encryption_sdk
```

Comment:
Is there an example of using this in the docs? The documentation you linked to doesn't even seem to mention a Python library. Is this official?

Reply:
Yes, I came to it by way of : https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/kms/client/generate_data_key.html

Can't like directly to the relevant part but it says:
>We recommend that you use the following pattern to encrypt data locally in your application. You can write your own code or use a client-side encryption library, such as the [Amazon Web Services Encryption SDK](https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/), the [Amazon DynamoDB Encryption Client](https://docs.aws.amazon.com/dynamodb-encryption-client/latest/devguide/), or [Amazon S3 client-side encryption](https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingClientSideEncryption.html) to do these tasks for you.